content/en/account_management/org_settings/cross_org_visibility.md

```diff
@@ -0,0 +1,176 @@
+        "relationships": {
+            "sink_org": {
+                "data": {
+                    "type": "orgs",
+                    "id": "{{the destination organization UUID}}"
+                }
+            }
+        }
+    }
+}
+{{< /code-block >}}
+
+#### Failure scenarios
+
+- The connection already exists
+- The connection refers to a destination organization ID outside of the account
+
+### Delete a connection
+
+Deletes a connection. Perform this operation either from the source organization or the destination organization. Reference the connection to delete with its ID, which you can get from the [List connections](#list-connections) request.
+
+<span style="padding:3px" class="font-semibold text-api-delete bg-bg-api-delete">DELETE</span> https://{datadog_site}/api/v2/org_connections/{connection_id}?api_key={datadog_api_key}&application_key={datadog_application_key}
+
+#### Failure scenarios
+
+- The organization does not participate as a source or a destination to the connection
+- The connection does not exist
+
+## Create a widget with cross-organization data
+
+Cross-organization [dashboard widgets][2] are available for Datadog organizations that exist as a _destination_ organization for at least one [connection](#configure-connections).
+
+Each query in a widget can display data from a single organization. You can combine queries into a cross-organization formula query.
+
+### In the UI
+
+Dashboard widgets allow you to create cross-organization queries when the following conditions are met:
+
+- You have cross-organization visibility enabled in your organization
+- At least one connection exists where the current organization is the destination
+
+If the previous conditions are true, an organization drop-down selector appears between the drop-down menus for data type and metric name. Use the organization drop-down selector to chose a source organization for your query.
+
+The following screenshot shows an example of a cross-organization formula query. The widget graphs the number of ingested events per service. To get the total number of events, the cross-organization formula query sums the data from organization A (in the query **a**) and organization B (in the query **b**).
+
+{{< img src="account_management/org_settings/cross_org_visibility/cross_org_query.png" alt="Screenshot showing configuration of a Dashboard widget with a cross-organization query" >}}
+
+### In the API
+
+You can define cross-organization queries in the following endpoint:
+- [Timeseries][8]
+
+When you define dashboard widgets in the dashboard API, use the `cross_org_uuids` parameter in the JSON widget definition payload to identify the source organization in a cross-organization query.
+
+The `cross_org_uuids` parameter is optional. If you omit `cross_org_uuids`, the query runs on the same organization in which you defined the widget.
+
+The [Datadog Terraform Provider][9] does not support cross-organization queries.
```

Comment:
Depending on how popular this is, consider making this an alert and move to the top of the section?
```suggestion
<div class="alert alert-info">
The <a href="https://registry.terraform.io/providers/DataDog/datadog/latest/docs">Datadog Terraform Provider</a> does not support cross-organization queries.
</div>
```